### PR TITLE
docs: watchtower guide + bump default sweep fee rate

### PR DESCRIPTION
**Pull request by cfromknecht**
This PR adds documentation for setting up watchtowers and configuring watchtower clients, and also resolves some logging issues related to double-hex encoded channel id's. It also bumps the default sweep rate from 3 sat/byte to 10, which provides a more comfortable default; the value can always be modified via `wtclient.sweep-fee-rate`.

### CODE REVIEW

File: docs/watchtower.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
+
+# Private Altruist Watchtowers
+
+As of v0.7.0,`lnd` supports the ability to run a private, altruist watchtower as
```
Comment (wpaulino) [inline docs/watchtower.md]: Nit: missing space after first comma.

File: docs/watchtower.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+a fully-integrated subsystem of `lnd`. Watchtowers act as a second line of
+defense in responding to malicious or accidental breach scenarios in the event
+that client’s node is offline or unable to respond at the time of a breach,
```
Comment (wpaulino) [inline docs/watchtower.md]: Nit: "that the client's"?

File: docs/watchtower.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+For now, no information regarding the operation of the watchtower client is
+exposed over the RPC interface. We are working to expose this information in a
+later release, progress on this can be tracked [in this
+PR](https://github.com/lightningnetwork/lnd/pull/3184). Users will be reliant on
+WTCL logs for observing the behavior of the client. We also plan to expand on
+the initial feature set by permitting multiple active towers for redundancy, as
+well as modifying the chosen set of towers dynamically without restarting the
+daemon.
+
+
```
Comment (wpaulino) [inline docs/watchtower.md]: Nit: extra newlines.

File: sample-lnd.conf [hunk 4]
```diff
@@ -340,3 +340,51 @@ litecoin.node=ltcd
 ; This means that multiple applications (other than lnd) using Tor won't be mixed
 ; in with lnd's traffic.
 ; tor.streamisolation=1
+
+[watchtower]
+; Enable integrated watchtower listening on :9911 by default.
+; watchtower.active=1
+
+; Specify the interfaces to listen on for watachtower client connections.  One
```
Comment (Roasbeef) [inline sample-lnd.conf]: watachtower -> watch tower

File: watchtower/wtclient/client_test.go [hunk 5]
```diff
@@ -1058,8 +1058,8 @@ var clientTests = []clientTest{
 		// less expensive than one that sweeps both.
 		name: "send and recv",
 		cfg: harnessCfg{
-			localBalance:  10000001, // ensure (% amt != 0)
-			remoteBalance: 20000001, // ensure (% amt != 0)
+			localBalance:  100000001, // ensure (% amt != 0)
```
Comment (Roasbeef) [inline watchtower/wtclient/client_test.go]: Typo in commit message: defasult -> default

**Comment by cfromknecht** [comment]
@Roasbeef @wpaulino comments addressed, ptal

**Pull request merged**